lib/ansible_tower_client/base_models/workflow_job_node.rb

```diff
@@ -8,11 +8,21 @@ def job
       api.jobs.find(job_id) if job?
     end
 
+    def workflow_job
```

Comment:
what calls this method?

Reply:
This is a public interface, the equivalent of `#job` up on line 7 but for `workflow_job` types